tests/test_mem_eff_attention.py

```diff
@@ -190,6 +190,7 @@ def _generate_op_device_dtype_biasT_B_Mq_Mkv_H_K_Kv(
                 for dtype in op.SUPPORTED_DTYPES:
                     bias_type = r.choice(LIST_MASKS)
                     # Avoid using too much memory
+                    B, Mq, Mkv, H, K, Kv = shape
```

Comment:
The original version of this line 5 lines below here should be deleted. (github doesn't let me comment outside the changes )